defaults/main.yml

```diff
@@ -17,6 +17,7 @@ nginx_service_state: started
 
 # Connection properties
 nginx_workers: auto
+#nginx_worker_rlimit_nofile --if you don't set FD's then OS settings will be used which is by default 2000
```

Comment:
IMHO the example should declare a valid value

Reply:
I had doubts about that when I was setting the variable. By default if the variable is not set, nginx get the value from OS settings http://nginx.org/en/docs/ngx_core_module.html#worker_rlimit_nofile

Finally I decided to replicate this behaviour with a conditional (https://github.com/idealista/nginx-role/pull/33/files#diff-754bfbe9e8e9c8a0ca748ca86030662e)

if you consider it necessary to assign a valid value I could set 2000 (OS default value)

Reply:
My proposal :)

```
# if you don't set FD's then OS settings will be used which is by default 2000
# nginx_worker_rlimit_nofile: 2000 
```

Reply:
much more evident this way